mlshvch/hometask-5/3.sql

```diff
@@ -0,0 +1,3 @@
+SELECT CONCAT(last_name, '-', first_name)
+FROM actor
+WHERE first_name LIKE 'W%' OR last_name LIKE '%h';
```

Comment:
Note: if you don't need to care about case, you can use case-insensitive `ILIKE` instead of regular `LIKE`.